lib/shopify_transporter/exporters/magento/customer_exporter.rb

```diff
@@ -12,14 +12,21 @@ def initialize(store_id: nil, soap_client: nil, database_adapter: nil)
 
         def export
           $stderr.puts 'Starting export...'
-          base_customers.map do |customer|
-            $stderr.puts "Fetching customer: #{customer[:customer_id]}..."
-            customer.merge(address_list: customer_address_list(customer[:customer_id]))
+          puts '['
+          base_customers.each_with_index do |base_customer, i|
+            puts ',' unless i == 0
+            $stderr.puts "Fetching customer: #{base_customer[:customer_id]}..."
+            print '  ' + JSON.pretty_generate(with_attributes(base_customer), object_nl: "\n  ")
```

Comment:
Ouch.

Reply:
I know... but I don't think there's a better way to generate output this pretty.